packages/design-system/src/components/ai-command-bar/command-bar.tsx

```diff
@@ -0,0 +1,103 @@
+import * as Collapsible from "@radix-ui/react-collapsible";
+import * as Separator from "@radix-ui/react-separator";
+import { styled, theme } from "../../stitches.config";
+import { Grid } from "../grid";
+import { Button } from "../button";
+import { type ReactNode } from "react";
+
+type Props = {
+  children: React.ReactNode;
+  content?: React.ReactNode;
+  open: boolean;
+  onOpenChange: (open: boolean) => void;
+};
+
+const borderRadius = 22;
+
+const GridContainer = styled(Grid, {
+  gridTemplateColumns: `max-content 1fr`,
+  gridAutoColumns: "max-content",
+  gridAutoFlow: "column",
+  padding: theme.spacing[5],
+  backgroundColor: "#1D1D1D",
+  borderRadius,
+  variants: {
+    open: {
+      true: {
+        borderTopLeftRadius: 0,
+        borderTopRightRadius: 0,
+        borderTop: "1px solid #323232",
+      },
+    },
+  },
+});
+
+const CollapsibleRoot = styled(Collapsible.Root, {
+  color: theme.colors.foregroundContrastMain,
+  backgroundColor: "#1D1D1D",
+  borderRadius,
+  border: "1px solid #323232",
+  boxShadow:
+    "0px 5px 17px 0px rgba(0, 0, 0, 0.30), 0px 2px 7px 0px rgba(0, 0, 0, 0.10)",
+});
+
+const CollapsibleContent = styled(Collapsible.Content, {
+  width: "min-content",
+  minWidth: "100%",
+});
+
+export const CommandBar = (props: Props) => {
+  const open = props.content !== undefined && props.open;
+  const onOpenChange =
+    props.content !== undefined ? props.onOpenChange : undefined;
+
+  return (
+    <CollapsibleRoot open={open} onOpenChange={onOpenChange}>
+      <CollapsibleContent>{props.content}</CollapsibleContent>
+      <GridContainer gap={2} open={props.open}>
+        {props.children}
+      </GridContainer>
+    </CollapsibleRoot>
+  );
+};
+
+export const CommandBarTrigger = Collapsible.Trigger;
+
+CommandBarTrigger.defaultProps = {
```

Comment:
defaultProps has been deprecated in react 18

Reply:
I know, just toooo much work to overwrite every component to add a new prop. Looking for simple solution

Reply:
yeah, but we can't do that, you are changing the default behavior for all components using this component, its a global effect

Reply:
As I wrote I know :) Just to start review faster until I find better solution

Reply:
in such cases use todo statements so we don't assume this is how you wanted it to be